website/api/blockchain-integration/contracts/verifier.md

```diff
@@ -52,6 +52,14 @@ RISC Zero deploys and manages the `RiscZeroVerifierRouter` contract listed below
 This contract will have verifiers added for each release of the zkVM, and will have verifiers removed in the case of security vulnerabilities.
 You can find detailed information in the [version management design][version-management], including information about how to manage your own copy of the smart contracts, if your application requires it.
 
+## Verification Cost
+
+SNARK verification via the `verifyProof` function on the [RISCZeroGroth16Verifier][RISCZeroGroth16Verifier] contract costs around 215k gas on all EVM chains. However, you'll notice that a transaction calling `verify()` via the [RiscZeroVerifierRouter](#verifer-router) costs [~316k gas][verify-sepolia-etherscan]. 
```

Comment:
Sending a transaction calling `verify()` to the a verifier (e.g. the router or Groth16 verifier) doesn't really accomplish anything, since it's a view function that doesn't change state or log events. Does this 316k include the intrinsic cost (21k) for the transactions? If so, I think it is more relevant to report the number without intrinsic gas since the call will be part of an existing transaction (calldata costs for the seal are relevant though, if we can count them).